pkg/controller/injector/k8s_binding_injector.go

```diff
@@ -98,4 +64,18 @@ func makeSecret(binding *servicecatalog.Binding, cred *brokerapi.Credential) *v1
 			"password": []byte(cred.Password),
 		},
 	}
+	// TODO: don't hardcode this namespace. https://github.com/kubernetes-incubator/service-catalog/issues/162
```

Comment:
I think we can talk through enough of the basics of #162 shortly that we can remove the hardcoding here, and would prefer to do that before this goes in.  Game?